lang/python/python-pytest/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-pytest
+PKG_VERSION:=5.4.1
+PKG_RELEASE:=1
+
+PYPI_NAME:=pytest
+PKG_HASH:=84dde37075b8805f3d1f392cc47e38a0e59518fb46a431cfdaf7cf1ce805f970
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pytest
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=Python testing framework
+  URL:=https://docs.pytest.org/en/latest/
+  DEPENDS:= \
+	+python3-light \
+	+python3-atomicwrites \
```

Comment:
atomicwrites became a [Windows-only dependency](https://github.com/pytest-dev/pytest/blob/5.4.1/setup.py#L10) in [5.3.0](https://github.com/pytest-dev/pytest/blob/master/doc/en/changelog.rst#pytest-530-2019-11-19) (pytest-dev/pytest#6148).

Reply:
Fixed. thx